clients/ui/bff/internal/data/model_version_test.go

```diff
@@ -2,17 +2,18 @@ package data
 
 import (
 	"encoding/json"
+	"net/http"
+	"net/url"
+	"testing"
+
 	"github.com/brianvoe/gofakeit/v7"
 	"github.com/kubeflow/model-registry/ui/bff/internal/mocks"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
-	"net/http"
-	"net/url"
-	"testing"
 )
 
 func TestGetModelVersion(t *testing.T) {
-	gofakeit.Seed(0)
+	_ = gofakeit.Seed(0)
```

Comment:
The errcheck linter really didn't like that we weren't checking errors on these statements. This is an ugly hack so maybe I should try to configure it to ignore our test files for this linter.

Reply:
For those cases https://golangci-lint.run/usage/false-positives/#nolint-directive also works.